src/components/autocomplete/js/autocompleteController.js

```diff
@@ -639,7 +679,7 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
   /**
    * Clears the searchText value and selected item.
    */
-  function clearValue ($event) {
+  function clearValue (shouldClearText) {
```

Comment:
oops, that was me rebasing and didn't noticing the `$event` was added, should I append the variable instead?